### DISCUSSION TITLE
LZ RULE v4.5.6

### DISCUSSION THREAD

**Discussion by larsonzh** [larsonzh/amdwprprsct#75] [Announcements]
**Asuswrt-Merlin dual WAN port router policy routing service configuration tool**

*华硕梅林固件路由器外网双线路接入策略路由服务配置工具*

v4.5.6 (2024/8/13)
1. 修复两处函数名称错误。
2. 完善和优化代码。
3. 在 Web UI 中添加捐助支持页面。

<hr /><em>This discussion was created from the release <a href='https://github.com/larsonzh/amdwprprsct/releases/tag/v4.5.6'>LZ RULE v4.5.6</a>.</em>

**Comment by sumic** [comment]
不错，还在坚持更新。 :+1: :
